src/pages/Login/index.tsx

```diff
@@ -119,7 +120,37 @@ const LoginPage = ({ type }: ILoginPage): JSX.Element => {
       content = (<BusinessForm onSubmit={onSubmit}/>);
       break;
     case 'login':
-      content = null;
+      onSubmit = async (data: LoginFormData) => {
+        const email = data.email;
+        const password = data.password;
+        signInWithEmailAndPassword(auth, email, password)
+          .then(async (userCredential) => {
+            // Signed in
+            const user = userCredential.user;
+            const reqData = {
+              firebaseId: user.uid,
+              email: data.email,
+              password: data.password,
```

Comment:
We do not save the password in our database. You can remove this. Also, if you wanted to save it to a database, you need to encrypted it. The password here is not encrypted. This is a security issue. If you want to send the password to a database you should encrypted it in-transit and at rest.